death-valley-town/Content/translation/deathvalley/polish.json

```diff
@@ -17,28 +17,28 @@
+    "building.death-valley-town.deathvalley.dwelling4.description": "Nawiedzona Świątynia pozwala rekrutować cienie.",
+    "building.death-valley-town.deathvalley.dwelling4.name": "Nawiedzona Świątynia",
     "building.death-valley-town.deathvalley.dwelling5.description": "Siedziba Nocy pozwala rekrutować mroczne czarownice.",
     "building.death-valley-town.deathvalley.dwelling5.name": "Siedziba Nocy",
     "building.death-valley-town.deathvalley.dwelling6.description": "Sala Ciemności pozwala rekrutować żniwiarzy.",
     "building.death-valley-town.deathvalley.dwelling6.name": "Sala Ciemności",
-    "building.death-valley-town.deathvalley.dwelling7.description": "Świątynia Śmierci pozwala rekrutować Kościane Golemy.",
-    "building.death-valley-town.deathvalley.dwelling7.name": "Świątynia Śmierci",
+    "building.death-valley-town.deathvalley.dwelling7.description": "Kanion Śmierci pozwala rekrutować Kościane Golemy.",
+    "building.death-valley-town.deathvalley.dwelling7.name": "Kanion Śmierci",
     "building.death-valley-town.deathvalley.dwellingUpgrade1.description": "Ulepszony cmentarz pozwala rekrutować Grobowe bestie.",
     "building.death-valley-town.deathvalley.dwellingUpgrade1.name": "Ul. Cmentarz",
     "building.death-valley-town.deathvalley.dwellingUpgrade2.description": "Ulepszony kopiec grobowy pozwala rekrutować Łuczników grozy.",
     "building.death-valley-town.deathvalley.dwellingUpgrade2.name": "Ul. Kopiec grobowy",
     "building.death-valley-town.deathvalley.dwellingUpgrade3.description": "Ulepszona wieża kościelna pozwala rekrutować Przegniłych.",
     "building.death-valley-town.deathvalley.dwellingUpgrade3.name": "Ul. Wieża kościelna",
-    "building.death-valley-town.deathvalley.dwellingUpgrade4.description": "Ulepszony nawiedzony wąwóz pozwala rekrutować Banshee.",
-    "building.death-valley-town.deathvalley.dwellingUpgrade4.name": "Ul. Nawiedzony wąwóz",
+    "building.death-valley-town.deathvalley.dwellingUpgrade4.description": "Ulepszony Nawiedzona Świątynia pozwala rekrutować Banshee.",
+    "building.death-valley-town.deathvalley.dwellingUpgrade4.name": "Ul. Nawiedzona Świątynia",
     "building.death-valley-town.deathvalley.dwellingUpgrade5.description": "Ulepszona siedziba Nocy pozwala rekrutować Nocne czarownice.",
     "building.death-valley-town.deathvalley.dwellingUpgrade5.name": "Ul. Siedziba Nocy",
     "building.death-valley-town.deathvalley.dwellingUpgrade6.description": "Ulepszona sala Ciemności pozwala rekrutować Ponurzych żniwiarzy.",
     "building.death-valley-town.deathvalley.dwellingUpgrade6.name": "Ul. Sala Ciemności",
-    "building.death-valley-town.deathvalley.dwellingUpgrade7.description": "Ulepszona świątynia Śmierci pozwala rekrutować Duchowe Golemy.",
-    "building.death-valley-town.deathvalley.dwellingUpgrade7.name": "Ul. Świątynia Śmierci",
+    "building.death-valley-town.deathvalley.dwellingUpgrade7.description": "Ulepszona Kanion Śmierci pozwala rekrutować Duchowe Golemy.",
```

Comment:
Ulepszona -> Ulepszony

Reply:
Thank you! I will fix it in a further PR 🙂